mnelab/model.py

```diff
@@ -131,22 +131,36 @@ def find_events(self, stim_channel, consecutive=True, initial_event=True,
     @data_changed
     def events_from_annotations(self):
         """Convert annotations to events."""
-        events, _ = mne.events_from_annotations(self.current["data"])
+        events, mappings = mne.events_from_annotations(self.current["data"])
+        # Mappings for annots are swapped {str: int} should be {int: str}
+        mappings = dict((v, k) for k, v in mappings.items())
         if events.shape[0] > 0:
             self.current["events"] = events
-            self.history.append("events, _ = "
-                                "mne.events_from_annotations(data)")
+            self.current["event_mappings"] = mappings
+            self.history.append("events, event_mappings = "
```

Comment:
Should be `self.history.append("events, mappings = "` I think?

Reply:
Sorry, I was trying to keep different names for the function variable, the instance attribute name, and the history variable, but it's unclear. I've updated for a concise name across the three, there is no naming collisions, because they refer to a variable, an attribute, and a history string.

Reply:
I think the dict key `"event_mappings"` is OK, but I'd vote for the shorter name `mappings` both in the code as well as in the history.

Reply:
Otherwise I think this is ready, so please also add a changelog entry and we're good to go.